pkgs/development/python-modules/pybars3/default.nix

```diff
@@ -0,0 +1,38 @@
+{ python3
+, fetchPypi
+, lib
+, pymeta3
+, buildPythonPackage
+}:
+buildPythonPackage rec {
+  pname = "pybars3";
+  version = "0.9.7";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-ashH6QXlO5xbk2rxEskQR14nv3Z/efRSjBb5rx7A4lI=";
+  };
+
+  propagatedBuildInputs = [
+    pymeta3
+  ];
+
+  checkPhase = ''
+    runHook preCheck
+    ${python3.interpreter} tests.py
+    runHook postCheck
```

Comment:
Isn't this just running the unittests with the tests under tests/? Can't pytest run those?

Reply:
Neither pytest, nor `python -m unittest` can discover and run these tests due to an ambiguity in this repo about what `tests` refers to. Here's an example of what happens if we let the unittest tool discover and run these tests:

```
ImportError: 'tests' module incorrectly imported from '/build/pybars3-0.9.7/tests'. Expected '/build/pybars3-0.9.7'. Is this module globally installed?
```

Upstream [runs this `tests.py` file in CI](https://github.com/wbond/pybars3/blob/bd2ce03def75bb777da1aa1b0c3884e04382023e/.github/workflows/ci.yml#L39), so we think it's best to do the same thing ourselves.

Reply:
Marking this resolved. Please reopen if you think we should do something different.